cmd/prefetchfiles-nri-plugin/main.go

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2023. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/containerd/nydus-snapshotter/pkg/errdefs"
+	"io"
+	"net"
+	"net/http"
+
+	"strings"
+
+	"log/syslog"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+
+	"github.com/containerd/nri/pkg/api"
+	"github.com/containerd/nri/pkg/stub"
+	"github.com/containerd/nydus-snapshotter/pkg/fanotify"
+	"github.com/containerd/nydus-snapshotter/version"
+	"github.com/pelletier/go-toml"
+)
+
+const (
+	defaultEvents     = "RunPodSandbox"
+	defaultServerPath = "/usr/local/bin/optimizer-server"
+	defaultPersistDir = "/opt/nri/optimizer/results"
+)
+
+type PluginConfig struct {
+	Events     []string `toml:"events"`
+	ServerPath string   `toml:"server_path"`
+	PersistDir string   `toml:"persist_dir"`
```

Comment:
It appears that this entry is not used in your nri plugin.

Reply:
Removed it in commits fd34